simulator/src/main/java/simblock/settings/SimulationConfiguration.java

```diff
@@ -37,7 +37,7 @@ public class SimulationConfiguration {
    */
   //TODO not documented in markdown
   // TODO return to PoW
-  public static final String ALGO = "simblock.node.consensus.ProofOfWork";
+  public static final String ALGO = "simblock.node.consensus.ProofOfSpace";
```

Comment:
This should ideally be read from a config file or command line param, but ok for now.
Please add a todo for that though (create an issue and link it in the TODO).